front/app/modules/commercial/insights/hooks/useInsightsInputsCount.tsx

```diff
@@ -20,28 +19,17 @@ const useInsightsInputsCount = (
     IInsightsInputsCount | undefined | null | Error
   >(undefined);
 
-  const previousViewId = useRef<string>();
-  const previousQueryParameters = useRef<Partial<QueryParameters>>();
+  const stringifiedQueryParameters = JSON.stringify(queryParameters);
 
   useEffect(() => {
-    if (
-      viewId === previousViewId.current &&
-      isEqual(queryParameters, previousQueryParameters.current)
-    ) {
-      return;
-    }
-
-    previousViewId.current = viewId;
-    previousQueryParameters.current = queryParameters;
-
     const subscription = insightsInputsCountStream(viewId, {
-      queryParameters,
+      queryParameters: JSON.parse(stringifiedQueryParameters),
```

Comment:
I had to change this implementation a bit as it was breaking our Edit page input counts.